pkg/cosign/bundle/bundle.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"github.com/sigstore/rekor/pkg/generated/models"
+)
+
+type Bundle struct {
+	VerificationData
```

Comment:
What's the difference between `VerificationData` and `VerificationMaterial`? I wonder if we could come up with more descriptive names so it's more clear :)

Reply:
It might be better to port that comment to the upstream bundle, but these are terms being used by those protos.
Data is extra auxiliary data for verification (timestamps, rekor entries), but verification material is the direct material (key hint, certificates)

Reply:
I am gonna port these and other questions to the upstream bundle. Thanks.